ext/src/ruby_api/execution_error.rs

```diff
@@ -10,26 +10,55 @@ use magnus::{
 };
 use std::borrow::Cow;
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+#[magnus::wrap(class = "Bluejay::ExecutionError::ErrorLocation")]
+struct ErrorLocation {
+    line: usize,
+    column: usize,
+}
+
+impl ErrorLocation {
+    fn to_h(&self) -> Result<RHash, Error> {
+        let ruby_h = rhash_with_capacity(2);
+        ruby_h.aset("line", self.line)?;
+        ruby_h.aset("column", self.column)?;
+        Ok(ruby_h)
+    }
+}
+
 #[derive(Clone, Debug, PartialEq, Eq)]
 #[magnus::wrap(class = "Bluejay::ExecutionError")]
 pub struct ExecutionError {
     message: Cow<'static, str>,
     path: Option<Vec<String>>,
+    locations: Option<Vec<ErrorLocation>>,
 }
 
 impl ExecutionError {
-    pub fn new(message: impl Into<Cow<'static, str>>, path: Option<Vec<String>>) -> Self {
+    pub fn new(
+        message: impl Into<Cow<'static, str>>,
+        path: Option<Vec<String>>,
+        locations: Option<Vec<ErrorLocation>>,
+    ) -> Self {
         Self {
             message: message.into(),
             path,
+            locations,
         }
     }
 
     fn rb_new(args: &[Value]) -> Result<Self, Error> {
-        let args = scan_args::<(String,), (Option<Vec<String>>,), (), (), (), ()>(args)?;
+        let args = scan_args::<
+            (String,),
+            (Option<Vec<String>>, Option<Vec<ErrorLocation>>),
```

Comment:
I'm not quite there on this: 

```
error[E0277]: the trait bound `ruby_api::execution_error::ErrorLocation: magnus::try_convert::TryConvertOwned` is not satisfied
    --> ext/src/ruby_api/execution_error.rs:53:13
     |
53   |             (Option<Vec<String>>, Option<Vec<ErrorLocation>>),
     |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `magnus::try_convert::TryConvertOwned` is not implemented for `ruby_api::execution_error::ErrorLocation`
     |
     = help: the following other types implement trait `magnus::try_convert::TryConvertOwned`:
               (T0, T1)
               (T0, T1, T2)
               (T0, T1, T2, T3)
               (T0, T1, T2, T3, T4)
               (T0, T1, T2, T3, T4, T5)
               (T0, T1, T2, T3, T4, T5, T6)
               (T0, T1, T2, T3, T4, T5, T6, T7)
               (T0, T1, T2, T3, T4, T5, T6, T7, T8)
             and 30 others
     = note: required for `std::vec::Vec<ruby_api::execution_error::ErrorLocation>` to implement `magnus::TryConvert`
     = note: required for `(std::option::Option<std::vec::Vec<std::string::String>>, std::option::Option<std::vec::Vec<ruby_api::execution_error::ErrorLocation>>)` to implement `magnus::scan_args::private::ScanArgsOpt`
     = note: required for `(std::option::Option<std::vec::Vec<std::string::String>>, std::option::Option<std::vec::Vec<ruby_api::execution_error::ErrorLocation>>)` to implement `magnus::scan_args::ScanArgsOpt`
note: required by a bound in `magnus::scan_args::scan_args`
    --> /Users/rmosolgo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/magnus-0.5.4/src/scan_args.rs:1198:10
     |
1193 | pub fn scan_args<Req, Opt, Splat, Trail, Kw, Block>(
     |        --------- required by a bound in this function
...
1198 |     Opt: ScanArgsOpt,
     |          ^^^^^^^^^^^ required by this bound in `scan_args`

```

Reply:
Ok, cool, thanks for that suggestion. It seems like the last step on this little part is to make it so that `ErrorLocation` can be created from a Hash. After an update, one of the Ruby tests is failing with: 

```
  1) Error:
Bluejay::TestSchema#test_execute_custom_scalar_coerce_result_error:
TypeError: no implicit conversion of Hash into Bluejay::ExecutionError::ErrorLocation
    /Users/rmosolgo/code/bluejay-rb/test/bluejay/test_schema.rb:304:in `new'
    /Users/rmosolgo/code/bluejay-rb/test/bluejay/test_schema.rb:304:in `test_execute_custom_scalar_coerce_result_error'
```

I'm _thinking_ this has to do with `TryConvert`, is that right? I haven't quite figure out what to _do_ with that, though...